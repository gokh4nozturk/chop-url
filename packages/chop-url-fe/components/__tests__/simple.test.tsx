import { render, screen } from '@testing-library/react';
import { describe, expect, it } from 'vitest';

describe('Simple Test Suite', () => {
  it('renders a simple div', () => {
    const testMessage = 'Hello Test';
    render(<div>{testMessage}</div>);

    const element = screen.getByText(testMessage);
    expect(element).toBeInTheDocument();
    expect(element.tagName).toBe('DIV');
  });

  it('matches snapshot', () => {
    const { container } = render(<div>Hello Test</div>);
    expect(container).toMatchSnapshot();
  });
});
