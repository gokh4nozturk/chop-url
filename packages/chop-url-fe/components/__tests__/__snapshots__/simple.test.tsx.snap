// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Simple Test Suite > matches snapshot 1`] = `
<div>
  <div>
    Hello Test
  </div>
</div>
`;
